testsuite/gnat2goto/tests/enum/enum.adb

```diff
@@ -0,0 +1,13 @@
+procedure Enum is
+  type E is (A, B);
+
+  function Get_A return E is (A);
+  function Get_B return E is (B);
+begin
+  --  CBMC Success
+  pragma Assert (A = Get_A);
+  --  CBMC Success
+  pragma Assert (B = Get_B);
+  --  CBMC Success
+  pragma Assert (Get_A /= Get_B);
```

Comment:
I don't like that this test doesn't have a failing case, the way cbmc works "absurd" code like infinite loops makes assertions pass as well, so if _all_ your assertions succeed you don't really know if your code is even reachable

Reply:
+1 yes.